src/nrtk_explorer/app/ui/__init__.py

```diff
@@ -0,0 +1,20 @@
+from .layout import build_layout
+from .image_list import image_list_component
+from .collapsible_card import card
+
+
+def reload(m=None):
+    from . import collapsible_card, image_list, layout
+
+    collapsible_card.__loader__.exec_module(collapsible_card)
+    image_list.__loader__.exec_module(image_list)
+    layout.__loader__.exec_module(layout)
+    if m:
+        m.__loader__.exec_module(m)
```

Comment:
Why is `exec_module` needed here, there is no instruction outside of the function in those modules.

Reply:
it is to enable hot reload from the outide